client/src/util/fetchFromAPI.test.js

```diff
@@ -1,13 +1,14 @@
 import { expect, test, it } from "@jest/globals";
+
 import { queryAPI } from "./fetchFromAPI";
 //reset the fetch mock so the previous tests fon't interfere with current tests
 beforeEach(() => {
   fetch.resetMocks();
 });
 
-it("mocks the fetch request", async () => {
+it("mocks the fetch request", async (options) => {
   fetch.mockResponseOnce(JSON.stringify({ foodMenu: { title: "potato" } }));
-  const response = await queryAPI();
+  const response = await queryAPI(options);
```

Comment:
Don't forget that `options` is an object that you need to create each time you use `queryAPI`.  It has required properties such as `url`, `onSuccess`, and `onError`.